explain-index-merge.md

```diff
@@ -5,41 +5,95 @@ summary: Learn about the execution plan information returned by the `EXPLAIN` st
+
++--------------------------------+---------+-----------+-------------------------+---------------------------------------------+
+| id                             | estRows | task      | access object           | operator info                               |
++--------------------------------+---------+-----------+-------------------------+---------------------------------------------+
+| IndexLookUp_19                 | 1.11    | root      |                         |                                             |
+| ├─IndexRangeScan_16(Build)     | 10.00   | cop[tikv] | table:t, index:idx_c(c) | range:[1,1], keep order:false, stats:pseudo |
+| └─Selection_18(Probe)          | 1.11    | cop[tikv] |                         | gt(test.t.a, 1), gt(test.t.b, 1)            |
+|   └─TableRowIDScan_17          | 10.00   | cop[tikv] | table:t                 | keep order:false, stats:pseudo              |
++--------------------------------+---------+-----------+-------------------------+---------------------------------------------+
+
+EXPLAIN SELECT /*+ USE_INDEX_MERGE(t, idx_a, idx_b, idx_c) */ * FROM t WHERE a > 1 AND b > 1 AND c = 1;  -- Uses index merge
++-------------------------------+---------+-----------+-------------------------+------------------------------------------------+
+| id                            | estRows | task      | access object           | operator info                                  |
++-------------------------------+---------+-----------+-------------------------+------------------------------------------------+
+| IndexMerge_9                  | 1.11    | root      |                         | type: intersection                             |
+| ├─IndexRangeScan_5(Build)     | 3333.33 | cop[tikv] | table:t, index:idx_a(a) | range:(1,+inf], keep order:false, stats:pseudo |
+| ├─IndexRangeScan_6(Build)     | 3333.33 | cop[tikv] | table:t, index:idx_b(b) | range:(1,+inf], keep order:false, stats:pseudo |
+| ├─IndexRangeScan_7(Build)     | 10.00   | cop[tikv] | table:t, index:idx_c(c) | range:[1,1], keep order:false, stats:pseudo    |
+| └─TableRowIDScan_8(Probe)     | 1.11    | cop[tikv] | table:t                 | keep order:false, stats:pseudo                 |
++-------------------------------+---------+-----------+-------------------------+------------------------------------------------+
+```
+
+From the preceding example, you can see that the filter condition is a `WHERE` clause that uses `AND` as the connector. Before index merge is enabled, the optimizer can only choose one of the three indexes (`idx_a`, `idx_b`, or `idx_c`).
+
+If one of the filter conditions has a good filtering performance, the optimizer directly chooses the corresponding index to achieve the ideal execution efficiency. However, if the data distribution meets all of the following three conditions, the optimizer considers using the intersection-type index merge:
```

Comment:
About the last sentence, considering that now we require the user to use the hint to enable the intersection type index merge, actually it's the user instead of the optimizer to consider if it's a suitable scenario.

Reply:
addressed